packages/framework/src/Controller/Admin/MailController.php

```diff
@@ -46,228 +35,100 @@ class MailController extends AdminBaseController
-            $selectedDomainId
-        );
-        $personalDataAccessTemplate = $this->mailTemplateFacade->get(
-            MailTemplate::PERSONAL_DATA_ACCESS_NAME,
-            $selectedDomainId
-        );
-
-        $personalDataExportTemplate = $this->mailTemplateFacade->get(
-            MailTemplate::PERSONAL_DATA_EXPORT_NAME,
-            $selectedDomainId
-        );
-
-        return [
-            'orderStatusesIndexedById' => $this->orderStatusFacade->getAllIndexedById(),
-            'orderStatusMailTemplatesByOrderStatusId' => $orderStatusMailTemplatesByOrderStatusId,
-            'orderStatusVariables' => $orderStatusesTemplateVariables,
-            'orderStatusVariablesLabels' => $this->getOrderStatusVariablesLabels(),
-            'registrationMailTemplate' => $registrationMailTemplate,
-            'registrationVariables' => $registrationTemplateVariables,
-            'registrationVariablesLabels' => $this->getRegistrationVariablesLabels(),
-            'resetPasswordMailTemplate' => $resetPasswordMailTemplate,
-            'resetPasswordRequiredVariables' => $resetPasswordTemplateRequiredVariables,
-            'resetPasswordVariables' => $resetPasswordTemplateVariables,
-            'resetPasswordVariablesLabels' => $this->getResetPasswordVariablesLabels(),
-            'TYPE_NEW' => OrderStatus::TYPE_NEW,
-            'personalDataAccessTemplate' => $personalDataAccessTemplate,
-            'personalDataAccessVariables' => $this->personalDataAccessMail->getSubjectVariables(),
-            'personalDataAccessRequiredVariablesLabels' => $this->personalDataAccessMail->getRequiredBodyVariables(),
-            'personalDataAccessVariablesLabels' => $this->getPersonalDataAccessVariablesLabels(),
-            'personalDataExportTemplate' => $personalDataExportTemplate,
-            'personalDataExportVariables' => $this->personalDataExportMail->getSubjectVariables(),
-            'personalDataExportRequiredVariablesLabels' => $this->personalDataExportMail->getRequiredBodyVariables(),
-            'personalDataExportVariablesLabels' => $this->getPersonalExportVariablesLabels(),
-        ];
+        return $this->render('@ShopsysFramework/Admin/Content/Mail/list.html.twig', [
+            'gridView' => $grid->createView(),
+        ]);
     }
 
     /**
-     * @Route("/mail/template/")
+     * @Route("/mail/edit/{id}", requirements={"id" = "\d+"})
      * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param int $id
+     * @param \Shopsys\FrameworkBundle\Model\AdminNavigation\BreadcrumbOverrider $breadcrumbOverrider
+     * @param \Shopsys\FrameworkBundle\Model\Mail\MailTemplateConfiguration $mailTemplateConfiguration
+     * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function templateAction(Request $request)
+    public function editAction(Request $request, int $id, BreadcrumbOverrider $breadcrumbOverrider, MailTemplateConfiguration $mailTemplateConfiguration): Response
```

Comment:
Yes, it's the same as injecting values in constructor, but the class property is not needed.
But we don't use this style of injecting dependencies and now it can be confusing. Thanks for pointing it out
Changed to constructor injection in 703abf604 and 55c1633c8